src/main/java/com/puppycrawl/tools/checkstyle/api/FileText.java

```diff
@@ -230,12 +229,7 @@ private static String readFile(final File inputFile, final CharsetDecoder decode
      */
     public static FileText fromLines(File file, List<String> lines) {
```

Comment:
@rnveach
I think, that there was sense in static factory method before @timurt's changes, but you are right, we do not need the method anymore.

@timurt
Please, remove the method and replace its usage with direct constructor call.